litewallet/ViewModels/Transaction.swift

```diff
@@ -36,18 +36,29 @@ class Transaction {
 		{
 			opsAmount = opsOutput.amount
 		}
-
-		self.fee = fee + opsAmount
+        
+        // Safely calculate fee = fee + opsAmount
```

Comment:
While comments like `// Safely calculate fee = fee + opsAmount` provide context, they are overly verbose and repeat what the code already expresses.
For example; `// Ensure the new fee calculation doesn't overflow`